standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Text.Json.xml

```diff
@@ -1064,10 +1075,10 @@ There is remaining data in the stream.</exception>
       <summary>A JSON array.</summary>
     </member>
     <member name="F:System.Text.Json.JsonValueKind.False">
-      <summary>Indicates that a value is the JSON value <see langword="false" />.</summary>
+      <summary>The JSON value false.</summary>
```

Comment:
Looks like we lose the tags like `<b>`. Maybe that's OK, but I am not sure.

https://github.com/dotnet/dotnet-api-docs/blob/9ea0edef8fe9034aee6b811dda7f6ee4ee9f55a7/xml/System.Text.Json/JsonValueKind.xml#L57